include/ebpf_api.h

```diff
@@ -176,6 +176,30 @@ extern "C"
         const char** error_message,
         ebpf_api_verifier_stats_t* stats);
 
+    /**
+     * @brief Convert an eBPF program to human readable byte code.
+     * @param[in] file Name of ELF file containing eBPF program.
```

Comment:
In the "from_memory" api, why do we need a name of "ELF file"?   What if there is no ELF file?